tests/e2e_test.py

```diff
@@ -5,10 +5,27 @@
 
 
 def get_cli_args():
-    parser = argparse.ArgumentParser(description=f"Script for running end to end tests for nxc")
-    parser.add_argument("-t", "--target", dest="target", required=True)
-    parser.add_argument("-u", "--user", "--username", dest="username", required=True)
-    parser.add_argument("-p", "--pass", "--password", dest="password", required=True)
+    parser = argparse.ArgumentParser(description="Script for running end to end tests for nxc")
+    parser.add_argument(
+        "-t",
+        "--target",
+        dest="target",
+        required=True
+    )
+    parser.add_argument(
+        "-u",
+        "--user",
+        "--username",
+        dest="username",
+        required=True
+    )
+    parser.add_argument(
+        "-p",
+        "--pass",
+        "--password",
+        dest="password",
+        required=True
+    )
```

Comment:
Would prefer to have this all as one-liner

Reply:
I much prefer the arguments to be separated. One liners for them make it a lot harder to read IMO.